invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -232,6 +232,14 @@ def _serialize_contributor(self, contributor):
             # (https://www.loc.gov/marc/bibliographic/bd700.html)
             affiliation = affiliations[0]["name"]
             contributor_dict["u"] = affiliation
+
+        role_id = contributor.get("role", {}).get("id", "")
+        if role_id:
+            props = get_vocabulary_props("contributorsroles", ["props.marc"], role_id)
```

Comment:
I'm merging this, but we'll need to get rid of this call again since it will kill performance

Reply:
got it, there is another call like this below for the record https://github.com/inveniosoftware/invenio-rdm-records/pull/1712